app/src/main/java/com/example/fragrecview/ui/userinput/UserInputFragment.kt

```diff
@@ -20,7 +20,7 @@ class UserInputFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         val view = inflater.inflate(R.layout.fragment_user_input, container, false)
-        userInputViewModel = ViewModelProvider(this).get(UserInputViewModel::class.java)
+        userInputViewModel = ViewModelProvider(this)[UserInputViewModel::class.java]
 
```

Comment:
I saw the load fragment method in each fragment. It's not required I think so. Can we create this loadFragment() method inside the activity?